src/main/java/com/github/alina/sesion4_challenge/ChallengeEighteen.java

```diff
@@ -0,0 +1,24 @@
+package com.github.alina.sesion4_challenge;
+//Write a Java program where you declare an integer array with 5 elements. Use a for loop to initialize the array such that each element is equal to its index incremented by 1. Then use another for loop to decrement each element by 1. Print the array before and after the decrement operation using a for-each loop
+public class ChallengeEighteen {
```

Comment:
Please consider using numbers instead of letters to describe the number of the challenge. This might help for an easier reading of the classes. This comment stands for all the classes from this PR.

Reply:
Fixed